src/Workspaces/Core/Portable/CodeFixes/CodeFixProvider.cs

```diff
@@ -18,6 +18,7 @@ public abstract class CodeFixProvider
 
         /// <summary>
         /// Computes one or more fixes for the specified <see cref="CodeFixContext"/>.
+        /// All the <see cref="CodeFixContext.Diagnostics"/> of the given <paramref name="context"/> must be supported by this fixer, as indicated by its <see cref="FixableDiagnosticIds"/> property.
```

Comment:
:bulb: I would move this information to be part of the documentation for the `context` parameter. In addition, consider the following:

```csharp
/// <param name="context">
/// A <see cref="CodeFixContext"/> containing context information about the diagnostics
/// to fix. The context must only contain diagnostics with an <see cref="Diagnostic.Id"/>
/// included in the <see cref="FixableDiagnosticIds"/> for the current provider.
/// </param>
```